packages/fether-react/src/Send/TxForm/TxForm.js

```diff
@@ -46,6 +48,20 @@ class Send extends Component {
       token
     } = this.props;
 
+    const decorator = createDecorator(
```

Comment:
Instead of defining `const decorator`, define it as a class member. The final form error should go away.